jobsets/tests/simple-node.nix

```diff
@@ -0,0 +1,22 @@
+import ./make-test.nix ({ pkgs, ... }: {
+  name = "boot";
+  nodes = {
+    machine = { config, pkgs, ... }: {
+      imports = [ ../../modules/cardano-node.nix ];
+      services.cardano-node = {
+        enable = true;
+        testIndex = 0;
```

Comment:
Best to use `import modules/cardano-node-config.nix 0 null` under imports for now to avoid these settings be duplicated around